src/frontend/org/voltdb/dbmonitor/js/voltdb.admin.ui.js

```diff
@@ -2946,6 +3305,258 @@ function loadAdminPage() {
+                    '       <label id="errorSecretKeyValue" for="txtSecretKeyValue" class="error" style="display: none;"></label>' +
+                    '   </td>' +
+                    '   <td></td>' +
+                    '</tr>';
+            } else {
+                $('#txtSecretKey').attr("disabled", "disabled");
+            }
+        }
+        $('#tblAddNewImportProperty tr.headerProperty').after(exportProperties);
+
+        removeDuplicateImportProperty();
+        setDefaultImportProperty();
+    };
+
+    var removeDuplicateImportProperty = function () {
+        $('#tblAddNewImportProperty :input').each(function () {
+            if ($(this).val() == "brokers") {
+                removeDuplicateImport(this, "brokers");
+            } else if ($(this).val() == "procedure") {
+                removeDuplicateImport(this, "procedure");
+            } else if ($(this).val() == "topics") {
+                removeDuplicateImport(this, "topics");
+            } else if ($(this).val() == "app.name") {
+                removeDuplicateImport(this, "app.name");
+            } else if ($(this).val() == "stream.name") {
+                removeDuplicateImport(this, "stream.name");
+            } else if ($(this).val() == "access.key") {
+                removeDuplicateImport(this, "access.key");
+            } else if ($(this).val() == "secret.key") {
+                removeDuplicateImport(this, "secret.key");
+            } else if ($(this).val() == "region") {
+                removeDuplicateImport(this, "region");
+            }
+        });
+    };
+
+    var removeDuplicateImport = function (object, propertyName) {
+        var exportType = $('#txtImportType').val();
+        if (!$(object).hasClass("requiredImportProperty")) {
+            var val = $(':input:eq(' + ($(':input').index(object) + 1) + ')').val();
+            if ($(VoltDbAdminConfig.newImportStreamMinPropertyName[propertyName]).length) {
+                $(VoltDbAdminConfig.newImportStreamMinPropertyName[propertyName]).val(val);
+                $(".newImportStreamMinProperty").addClass("addedImportProperty");
+                var $row = $(object).closest("tr");
+                $row.remove();
+            } else if ($(VoltDbAdminConfig.newImportStreamMinPropertyName[propertyName + '_' + exportType]).length) {
+                $(VoltDbAdminConfig.newImportStreamMinPropertyName[propertyName + '_' + exportType]).val(val);
+                $(".newImportStreamMinProperty").addClass("addedImportProperty");
+                var $row1 = $(object).closest("tr");
+                $row1.remove();
+            }
+        }
+    };
+
+    var setDefaultImportProperty = function () {
+        var importType = $('#txtImportType').val();
+        if (importType.toUpperCase() == "KAFKA") {
+            setDefaultImportDisplay($("#txtBrokers"));
+            setDefaultImportDisplay($("#txtProcedure"));
+            setDefaultImportDisplay($("#txtTopics"));
+        } else {
```

Comment:
Whats the difference between if and else here can this be done in a function that takes flag